### PR TITLE
Initial file for githubernie bio

### PR DESCRIPTION
**Pull request by ernie-m**
These are more details on my initial file creation for githubernie bio file.  this is a really simple change, because it is a really simple file...so what did you expect?

### CODE REVIEW

File: bios/githubernie-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Heading of my file
+- I live in buenos aires
```
Comment (ernie-m) [inline bios/githubernie-bio.md]: buenos aires is in argentina right?

Comment (ernie-m) [inline bios/githubernie-bio.md]: that's a great ideal...i'll add it now

File: bios/githubernie-bio.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Heading of my file
+- I live in buenos aires, which is in argentina
+- ok, no i really don't
+- i live in SD
```
Comment (lwdupont) [inline bios/githubernie-bio.md]: North or south of work? :)

Comment (ernie-m) [inline bios/githubernie-bio.md]: south, i would be way too bored living anywhere north of the office :)

**Pull request merged**